### DISCUSSION TITLE
Enabling/deleting revisions per document

### DISCUSSION THREAD

**Discussion by astalegale** [ravendb/ravendb#13747] [Q&A] [answered]
Hi,
 i wonder if it is possible to enable/delete the revisions per document only.

We have a situation where not all the document in a collection need to have revision but for logic needing we need to change its states.

Thanks
Alex

**Comment by ayende** [comment, marked as answer]
Yes, see:

https://github.com/ravendb/ravendb/blob/8a0fcd6751fa98863583fbc20409eaddefbb6191/test/FastTests/Server/Documents/Revisions/RevisionsTests.cs#L1801

And usage:

    await store.Maintenance.SendAsync(new DeleteRevisionsOperation(new AdminRevisionsHandler.Parameters {DocumentIds = ids}));


Note that this must be run as a db admin at least to work.

**Reply by astalegale** [reply]
Many thanks for the hint

Regards

Alex
